.github/workflows/archive_lint_reports.yml

```diff
@@ -0,0 +1,46 @@
+name: Copy Result JSON on Commit
+
+on:
+  push:
+    branches:
+      - did-lint-reports
+    paths:
+      - 'result.json'
+
+jobs:
+  copy-result-json:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      
+      - name: Get current timestamp
+        id: timestamp
+        run: echo "::set-output name=timestamp::$(date +'%Y%m%d%H%M%S')"
```

Comment:
`:: set-output` is deprecated. While Github has paused removing it entirely, at some point, it might stop working at some point.

This is equivalent and would be currently supported:

```
echo "name=timestamp::$(date +'%Y%m%d%H%M%S')" >> "$GITHUB_OUTPUT"
```